arbnode/espresso_caff_node.go

```diff
@@ -183,6 +193,8 @@ func NewEspressoCaffNode(
 	sequencerInbox *SequencerInbox,
 	fatalErrChan chan error,
 	httpPort int,
+	databaseParentDir string,
+	l2ChainDataDir string,
```

Comment:
so many arguments, why don't we pass in the `stack *node.Node` in place of `stack.Config().HTTPPort`, then these dir info can be retrieved from `stack.DataDir()`

Reply:
true! Lets do that